oojs/application.js

```diff
@@ -1,13 +1,23 @@
 $(document).ready(function() {
-  $('#roller button.add').on('click', function() {
-    console.log("WAT")
-    $('.dice').append('<div class="die">0</div>');
-  });
-
-  $('#roller button.roll').on('click', function() {
-    $('.die').each(function(k, die) {
-      var value = Math.floor((Math.random()*6)+1);
-      $(die).text(value);
+    var dice = {};
+
+    dice.addDie = function() {
```

Comment:
You can move these out of the `ready` function since they don't need to wait for render, you can also put these into classes as well.